web/modules/custom/pineapple_general/pineapple_general.module

```diff
@@ -33,8 +33,8 @@ function pineapple_general_field_widget_form_alter(&$element, \Drupal\Core\Form\
   // If paragraph type exists and its value is equal to image_paragraph.
   if (isset($element['#paragraph_type']) && $element['#paragraph_type'] == 'image_paragraph') {
     $user_roles = \Drupal::currentUser()->getRoles();
-    // If the hero image field exists and if content editor is in the user roles array.
-    if (isset($element['subform']['field_p_ip_hero_image']) && in_array('content_editor', $user_roles)) {
+    // If the hero image field exists and if admin is not in the user roles array.
+    if (isset($element['subform']['field_p_ip_hero_image']) && !in_array('admin', $user_roles)) {
```

Comment:
Did you test this?

Reply:
Yes

Reply:
Yes, however I just saw it works inconsistently. I figured out the issue and should be fine now.